src/main/java/org/mitre/synthea/export/JSONExporter.java

```diff
@@ -0,0 +1,125 @@
+package org.mitre.synthea.export;
+
+import com.google.gson.ExclusionStrategy;
+import com.google.gson.FieldAttributes;
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+import com.google.gson.JsonElement;
+import com.google.gson.JsonObject;
+import com.google.gson.JsonPrimitive;
+import com.google.gson.JsonSerializationContext;
+import com.google.gson.JsonSerializer;
+
+import java.lang.reflect.Modifier;
+import java.lang.reflect.Type;
+import java.util.List;
+
+import org.mitre.synthea.engine.State;
+import org.mitre.synthea.helpers.Config;
+import org.mitre.synthea.world.agents.Payer;
+import org.mitre.synthea.world.agents.Person;
+
+/**
+ * Class that manages the export of a Synthea Person and their related information into a JSON
+ * format. This is a somewhat "raw" export, as the information is only lightly edited before export.
+ * In other words, it's pretty close to an object dump.
+ */
+public class JSONExporter {
+
+  /**
+   * Export the given Person object into a String full of JSON.
+   * @param person to export
+   * @return a lot of JSON in a String
+   */
+  public static String export(Person person) {
+    Gson gson = new GsonBuilder()
```

Comment:
It might be nice to enable pretty printing on the resulting JSON, otherwise it all gets written to one line:
```suggestion
    Gson gson = new GsonBuilder()
            .setPrettyPrinting()
```